### ISSUE TITLE
spin error WebpackOptionsValidationError

### ISSUE THREAD

**Issue by sis-dk** [sysgears/create-apollo-app#56]
**What I did?**
- git clone https://github.com/sysgears/create-apollo-app.git
- yarn create apollo-app my-app@mobile
- cd my-app/
- yarn start

**What the problem is?**
- After yarn start, it throws this error,

```
yarn run v1.9.4
$ spin start
spin error WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.module.rules[0] has an unknown property 'type'. These properties are valid:
   object { enforce?, exclude?, include?, issuer?, loader?, loaders?, oneOf?, options?, parser?, query?, resource?, resourceQuery?, compiler?, rules?, test?, use? }
   -> A rule
    at webpack (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/webpack/lib/webpack.js:19:9)
    at /home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:717:24
    at new Promise (<anonymous>)
    at buildDll (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:705:10)
    at runBuilder (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:1022:57)
    at Object.execute (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:1147:5)
    at Object.<anonymous> (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/cli.ts:42:12)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
(node:1752) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated
spin error WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.module.rules[0] has an unknown property 'type'. These properties are valid:
   object { enforce?, exclude?, include?, issuer?, loader?, loaders?, oneOf?, options?, parser?, query?, resource?, resourceQuery?, compiler?, rules?, test?, use? }
   -> A rule
    at webpack (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/webpack/lib/webpack.js:19:9)
    at /home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:717:24
    at new Promise (<anonymous>)
    at buildDll (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:705:10)
    at runBuilder (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:1022:57)
    at Object.execute (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/executor.ts:1147:5)
    at Object.<anonymous> (/home/sisir/Atomic-Design/create-apollo-app/my-app/node_modules/spinjs/src/cli.ts:42:12)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
(node:1744) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated
```

**Comment by vlasenko** [comment]
@sis-dk Should be fixed in `spinjs@0.4.156`. 

There is no need to clone `create-apollo-app`.

Please make sure you remove `my-app` before trying again (so that latest `spinjs` was installed).

It is sufficient to only do these steps:
  - yarn create apollo-app my-app@mobile
  - cd my-app/
  - yarn start

**Comment by sis-dk** [comment]
@vlasenko followed your instructions and it works now. Thanks.
